contracts/PoaOperator.sol

```diff
@@ -335,9 +344,11 @@ contract PoaOperator is Adminable {
     // check time
     require(now >= chal.endTime, "time not expired yet");
     // transfer funds
-    chal.challenger.transfer(CHAL_STAKE);
+    chal.challenger.transfer(vault.exitStake());
     // delete period
     bridge.deletePeriod(_period);
   }
 
+  // solium-disable-next-line mixedcase
+  uint256[18] private ______gap;
```

Comment:
why is this needed? 🤔

Reply:
we are using this in other contracts as well. if contract that inherit from this one have storage, and we want to add params to base contract, then this param can be used to set off the shift in storage layout.

Reply:
aren't we already covering this in Initializable? https://github.com/leapdao/leap-contracts/blob/master/contracts/zos/Initializable.sol#L56

Reply:
ah, 50 slots in Initializable allow us to add things to initializable, right? But we are not extending PoaOperator yet to add this gap, no?

Reply:
i'm proposing it here, so if we add storage to PosOperator later, we are not stuck. but you are right. out of scope.

Reply:
good point. Missed the fact that PosOperator extends PoaOperator 👍

Reply:
`challenges` mapping is not capped. Should we reserve more than 18 memory slots here? If there are a lot of challenges their storage may overlap with PosOperator, no?

Reply:
> challenges mapping is not capped

i think every variable only takes 1 position. this is not at runtime, but when the compiler assigns an address to each variable, i think.